lms/djangoapps/certificates/views/webview.py

```diff
@@ -342,7 +343,7 @@ def _get_user_certificate(request, user, course_key, course, preview_mode=None):
     user_certificate = None
     if preview_mode:
         # certificate is being previewed from studio
-        if has_access(request.user, 'instructor', course) or has_access(request.user, 'staff', course):
+        if request.user.has_perm(PREVIEW_CERTIFICATES, course):
```

Comment:
Will this handle both instructors and staff (will it match the existing code)?

Reply:
@cpennington was saying that both get checked
i'm not sure if i'm following the code correctly but i think both get checked here regardless of the action https://github.com/edx/edx-platform/blob/master/lms/djangoapps/courseware/access.py#L402